tabs/src/components/app/views/Monaco.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import MonacoEditor from 'react-monaco-editor';
+import PropTypes from 'prop-types';
+
+
+const Monaco = ( props ) => {
+  let { body, height, onChange, readOnly } = props;
+
+  if (body && typeof body !== 'string') {
+    body = formatJsonStringForAllBrowsers(body);
+  }
+
+  return (
+    <MonacoEditor
+      width='5 !important'
```

Comment:
what's the `5` mean here 👀 5px, 5%, 5vw ?

Reply:
Oh wow, i didnt realize that I didnt change that. Thanks